responsemanager/responsemanager_test.go

```diff
@@ -291,6 +291,7 @@ func TestValidationAndExtensions(t *testing.T) {
 		// request fails with base loader reading from block store that's missing data
 		responseManager.ProcessRequests(td.ctx, td.p, td.requests)
 		td.assertCompleteRequestWith(graphsync.RequestFailedContentNotFound)
+		td.taskqueue.WaitForNoActiveTasks()
```

Comment:
since the timeout shows up in other subtests here I think it might be safest to just do this within `assertCompleteRequestWith()`